tests/dir_helpers.py

```diff
@@ -67,8 +67,7 @@
 
 
 # see https://github.com/iterative/dvc/issues/3167
-git_logger = logging.getLogger("git")
-git_logger.setLevel(logging.CRITICAL)
+disable_other_loggers()
```

Comment:
It'd be great if we could move this to conftests. Not super important at the moment, but it does not belong here.